src/components/Categories.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function Categories() {
+  return (
+    <div>Categories</div>
+  );
```

Comment:
- [ ] (Optional) It would be nice to follow the project requirements and render a `button` with text _**Check Status**_ for this component. 👍